### ISSUE TITLE
[a11y] when on a sortable column, should announce if it's sorted ascending or descending

### ISSUE THREAD

**Issue by Jacklynn** [18F/crt-portal#287]
Here's the documentation I provided on the [original issue](https://app.zenhub.com/workspaces/doj-crt-intake-scrum-board-5d03bf56c1c8a35d482eca0f/issues/18f/crt-portal/36) for what I would expect a sortable column to announce for screen reader users: https://dequeuniversity.com/library/aria/tables/sf-table-sortable
 
Basically: column name, and how it's sorted (ascending, descending)

**Comment by Jacklynn** [comment]
⬇️⬆️🗣🙌
